frontend/tests/gqlHelpers.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, type APIRequestContext } from '@playwright/test';
+import { serverBaseUrl } from './envVars';
+
+export function validateGqlErrors(json: {error: unknown, data: unknown}, expectError = false): void {
```

Comment:
I think it's plural
```suggestion
export function validateGqlErrors(json: {errors: unknown, data: unknown}, expectError = false): void {
```

Reply:
Fixed in commit 0e3162c0541f34f97a0d616d043df3a29060e1cb.